edx_sga/tests/integration_tests.py

```diff
@@ -779,8 +780,11 @@ def test_replace_url(self, has_static_asset_path):
         If the static asset path is set on a course, it should be substituted when the course is rendered
         """
         # make a runtime with a static asset path, which will override the base_asset_url
-
         static_asset_path = '/a/different/static/asset/path'
+        solution_expected = '/a/different/static/asset/path'
+        if django.VERSION >= (1, 11):
```

Comment:
I'm not sure this conditional is the right idea here. What caused the change of behavior in Django? Maybe we should be using an absolute path for the static root in the test settings instead